e2etest/bgp_test.go

```diff
@@ -94,69 +97,66 @@ var _ = ginkgo.Describe("BGP", func() {
 		}
 	})
 
-	ginkgo.Context("type=Loadbalancer", func() {
+	table.DescribeTable("set protocol loadbalancer", func(frrContainerIP *string, version string) {
 		var allNodes *corev1.NodeList
-		ginkgo.BeforeEach(func() {
-			configData := configFile{
-				Pools: []addressPool{
-					{
-						Name:     "bgp-test",
-						Protocol: BGP,
-						Addresses: []string{
-							"192.168.10.0/24",
-							"fc00:f853:0ccd:e799::/124",
-						},
+		configData := configFile{
+			Pools: []addressPool{
+				{
+					Name:     "bgp-test",
+					Protocol: BGP,
+					Addresses: []string{
+						"192.168.10.0/24",
+						"fc00:f853:0ccd:e799::/124",
 					},
 				},
-				Peers: []peer{
-					{
-						Addr:  frrContainerIPv4,
-						ASN:   64512,
-						MyASN: 64512,
-					},
+			},
+			Peers: []peer{
+				{
+					Addr:  *frrContainerIP,
+					ASN:   64512,
+					MyASN: 64512,
 				},
-			}
-			err := updateConfigMap(cs, configData)
-			framework.ExpectNoError(err)
+			},
+		}
+		err := updateConfigMap(cs, configData)
+		framework.ExpectNoError(err)
 
-			pairExternalFRRWithNodes(cs)
+		pairExternalFRRWithNodes(cs, version)
 
-			allNodes, err = cs.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
-			framework.ExpectNoError(err)
-		})
+		allNodes, err = cs.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+		framework.ExpectNoError(err)
 
-		ginkgo.It("should work for ExternalTrafficPolicy=Cluster", func() {
-			svc, _ := createServiceWithBackend(cs, f.Namespace.Name, corev1.ServiceExternalTrafficPolicyTypeCluster)
+		ginkgo.By("should work for ExternalTrafficPolicy=Cluster")
```

Comment:
Why this change? By doing this, before / after each won't run. I'd rather two tables / have the externalpolicy as another parameter of the table.